plugins/woocommerce-blocks/assets/js/previews/utils.ts

```diff
@@ -19,16 +19,18 @@ export const API_SITE_CURRENCY = {
 /**
  * Preview data is defined with 2dp. This converts to selected currency settings.
  */
-export const displayForMinorUnit = ( value: string ): string => {
+export const displayForMinorUnit = ( value: string, precision = 2 ): string => {
```

Comment:
Nitpic: Given that `value` has an explicit type, shouldn't we also provide an explicit type for `precision`?

```suggestion
export const displayForMinorUnit = ( value: string, precision: number = 2 ): string => {
```

I'm thinking about the case in which we call `displayForMinorUnit()` and the value of `precision` is not a number. By adding an explicit type, we can catch such an error during development.

Reply:
This does not appear to be needed. The type is implied by the default we set (2). Checked to confirm:

![Screenshot 2025-01-22 at 14 25 38](https://github.com/user-attachments/assets/ce276d06-3c07-4df9-a6c8-d502cb7c4c57)

Reply:
Thanks for clarifying this, @mikejolley. I was thinking about consistency and demonstrating best practices to the community, but you're right that the type is already implied by the default value. I'll mark this suggestion as resolved.